casper/src/main/scala/coop/rchain/casper/genesis/contracts/ProofOfStake.scala

```diff
@@ -6,6 +6,14 @@ import coop.rchain.crypto.codec.Base16
 //TODO: include other fields relevent to PoS (e.g. rewards channel)
 case class ProofOfStakeValidator(id: Array[Byte], stake: Long)
 
+case class ProofOfStakeParams(
+    minimumBond: Long,
+    maximumBond: Long,
+    validators: Seq[ProofOfStakeValidator]
+) {
+  require(minimumBond <= maximumBond)
```

Comment:
If we do `require` now, is `require(validators.nonEmpty)` also correct?

Reply:
Yeah, good call.